endpoints/api/superuser.py

```diff
@@ -103,7 +103,6 @@ def get(self, parsed_args):
 
 
 @resource("/v1/superuser/logs")
-@internal_only
```

Comment:
Why are we making this one not internal?

Reply:
It is plainly for docs. Gabriel requested for it. Currently they need to remember to add in this one manually every time. It would make a big difference as they can auto generate all the API docs with this change.